openproblems/tasks/cell_cell_communication/datasets/tnbc_wu2021.py

```diff
@@ -0,0 +1,14 @@
+from ....data.tnbc_wu2021 import load_tnbc_data
+from ....tools.decorators import dataset
+
+
+@dataset(
+    "~43k Cells from Triple Negative Breast Cancer"
+    "Wu et al., 2021. Nature genetics, 53(9), pp.1334-1347.",
```

Comment:
"43k Cells from Triple Negative Breast Cancer" should be sufficient given we link the reference

Reply:
Agreed.